bin/compile

```diff
@@ -57,6 +57,13 @@ head "Building dependencies"
 cd $build_dir
 build_dependencies
 
+head "Running tests"
+if [[ $(read_json "$build_dir/package.json" ".scripts.test") != "" ]]; then
+  npm test
+else
+  warning "No tests specified. Please do it ASAP!"
```

Comment:
yeah. all messages needs to be reviewed in buildpacks. for now there are a lot of heroku lines)